app/parsers/bulkrax/csv_parser.rb

```diff
@@ -202,25 +199,14 @@ def write_files
       end
     end
 
-    def key_allowed(key)
-      !Bulkrax.reserved_properties.include?(key) &&
-        new_entry(entry_class, 'Bulkrax::Exporter').field_supported?(key) &&
+    def export_key_allowed(key)
+      new_entry(entry_class, 'Bulkrax::Exporter').field_supported?(key) &&
         key != source_identifier.to_s
```

Comment:
is it a rails thing to have subsequent lines in a conditional be indented beyond the first line?

Reply:
yes this is proper spacing in rails